src/main/java/bio/terra/cli/command/config/List.java

```diff
@@ -55,6 +59,8 @@ public ConfigListReturnValue(
   private static void printText(ConfigListReturnValue returnValue) {
     OUT.println("[browser] browser launch for login = " + returnValue.browser);
     OUT.println("[image] docker image id = " + returnValue.image);
+    OUT.println(
```

Comment:
maybe `resource-cache-limit` would be more precise, and you might want to configure something else about resources at some point. It's more typing though.

Reply:
Changed to `resource-limit`.